src/main/java/seedu/address/model/time/TimePeriod.java

```diff
@@ -117,10 +118,10 @@ public List<HourBlock> fragmentIntoHourBlocks() {
 
     @Override
     public String toString() {
-        return String.format("[%s] %s - %s",
+        return String.format("[%s] %d - %d",
             getSchoolDay(),
-            TimeUtil.formatLocalTime(getStartTime()),
-            TimeUtil.formatLocalTime(getEndTime()));
+            getStartTime().getHourOfDay(),
+            getEndTime().getHourOfDay());
```

Comment:
`TimeUtils.formatLocalTime` gives you the AM and PM too. I think you should revert it back. I use it for debugging purposes and it is easier for me.